packages/vm/.eslintrc.js

```diff
@@ -1,18 +1,10 @@
 module.exports = {
   extends: "@ethereumjs/eslint-config-defaults",
-  ignorePatterns: ["tests", "scripts", "examples", "karma.conf.js"],
+  ignorePatterns: ["scripts", "examples", "karma.conf.js"],
   rules: {
-    '@typescript-eslint/await-thenable': 'off',
-    '@typescript-eslint/no-floating-promises': 'off',
-    '@typescript-eslint/no-unused-vars': 'off',
     '@typescript-eslint/no-use-before-define': 'off',
```

Comment:
If we enable this one, then ESLint will complain if you use a function which is not yet defined (even in "global" scope). This is stylish inside functions (do not call a function not yet declared), but it is not in utility files. For instance if a function in an utility file `A` calls into `B` and `B` calls into `A` then of course one of them will be defined after calling the other and hence ESLint throws. We can also disable this rule for these specific files.

Reply:
Yeah, maybe disabling for specific files is a good idea? I generally find this a useful and somewhat code-structure enforcing rule.

Reply:
We can do this in a follow-up PR, I think it were only util files which the linter didn't like.